### PR TITLE
Fix #112 - Remove gemini variations (prod)

### LINKED ISSUE #112: Remove gemini variations (prod)

### PR DESCRIPTION
**Pull request by jumde**
fix #112

### CODE REVIEW

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x64` via the following STR/Cases:

*using `--variations-server-url=https://variations.bravesoftware.com/seed`, launched Brave twice and ensured the following:

* `BraveRewardsGeminiNightly` wasn't being listed under `Variations` via `brave://version`
* ensured that both Uphold & Gemini were being displayed as possible custodian when verifying a rewards wallet

Builds used during QA:

* `1.30.59 Chromium: 93.0.4577.58 (Official Build) beta (64-bit)` - **PASSED**
* `1.31.12 Chromium: 93.0.4577.58 (Official Build) nightly (64-bit)` - **PASSED**

**`1.30.59 Chromium: 93.0.4577.58`** | **`1.31.12 Chromium: 93.0.4577.58`**
-----------------------------------------|--------------------------------------------
nightly | beta

**Comment by stephendonner** [comment]
Verification PASSED on macOS-Intel (x86) via the following STR/Cases:

*using `--variations-server-url=https://variations.bravesoftware.com/seed`, launched Brave twice and ensured the following:

    `BraveRewardsGeminiNightly` wasn't being listed under Variations via `brave://version`
    ensured that both Uphold & Gemini were being displayed as possible custodian when verifying a rewards wallet

Builds used during QA:

* `1.30.59 Chromium: 93.0.4577.58 (Official Build) beta (x86_64)` - PASSED
* `1.31.11 Chromium: 93.0.4577.58 (Official Build) nightly (x86_64)`

`nightly` | `beta`
---------|---------
Screen Shot 2021-09-01 at 10 33 00 AM |  Screen Shot 2021-09-01 at 10 35 44 AM


`nightly` | `beta`
---------|---------
Screen Shot 2021-09-01 at 10 39 23 AM | Screen Shot 2021-09-01 at 10 39 31 AM

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` via https://github.com/brave/brave-variations/pull/114#issuecomment-910498518 & https://github.com/brave/brave-variations/pull/114#issuecomment-910509009.

**Pull request merged**
